sync/sync-impl/src/main/java/com/duckduckgo/sync/impl/SyncAccountRepository.kt

```diff
@@ -34,7 +34,7 @@ import com.squareup.moshi.Moshi
 import dagger.SingleInstanceIn
 import javax.inject.*
 
-interface SyncRepository {
```

Comment:
if we are making changes here, can you please also add javadocs?

Reply:
This is `:impl` module, and not visible to other features. do we need them?